cluster/v1alpha1/types.go

```diff
@@ -0,0 +1,100 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:subresource:status
+// +kubebuilder:resource:scope="Cluster"
+
+// ManagedClusterSet is a set of managed clusters.
```

Comment:
What properties can we describe here that will be useful to know?

Reply:
Maybe we could add things like this
ManagedClusterSet defines a group of ManagedCluster that user's workload can run on. A workload can be defined to deployed on a ManagedClusterSet, which mean
1. the workload can run on any ManagedCluster in the ManagedClusterSet 
2. the workload cannot run on any ManagedCluster outside the ManagedClusterSet
3. the service exposed by the workload can be shared in any ManagedCluster in the ManagedClusterSet

Reply:
Comments updated.